File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height ");
    }
    while (n <= 0 || n > 8);

    for (int hash = 1; hash <= n; hash++)
    {
        int dots = n - hash;
        for (int dot = 0; dot < dots; dot++)
        {
            printf(" ");
        }
        for (int dot = 0; dot < hash; dot++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario-less/mario.py
from cs50 import get_int

def get_height():
    while True:
        try:
            h = int(input("height: "))
            if h > 0 and h < 9:
                return h
        except ValueError:
             pass


h = get_height()

for times in range(1, h+1):
        print(" "*(h-times) + "#"*times + "  " + "#"*times)







